tests/model/test_model.py

```diff
@@ -2079,20 +2095,66 @@ def test_toggle_stream_pinned_status(
                     "id": 0,
                 },
                 True,
+                False,
+                id="op_stop",
             ),
         ],
-        ids=["not_in_pm_narrow", "not_in_pm_narrow_with_sender", "start", "stop"],
     )
-    def test__handle_typing_event(self, mocker, model, narrow, event, called):
+    def test__handle_typing_event(
+        self,
+        mocker,
+        model,
+        narrow,
+        event,
+        is_animation_in_progress,
+        expected_is_recipient_typing,
+    ):
         event["type"] = "typing"
 
-        mocker.patch("zulipterminal.ui.View.set_footer_text")
+        mocker.patch("zulipterminal.model.Model._show_typing_animation")
         model.narrow = narrow
         model.user_dict = {"hamlet@zulip.com": {"full_name": "hamlet"}}
+        model.is_recipient_typing = False
+        model.is_typing_animation_in_progress = is_animation_in_progress
+        expect_indicator = expected_is_recipient_typing and not is_animation_in_progress
```

Comment:
It would be more explicit and readable if we put it as a parameter in the parametrization. Also, let's include `_show_typing_animation` in the name as with this we're essentially testing whether `_show_typing_animation` was called.